grammars.yml

```diff
@@ -442,6 +440,8 @@ vendor/grammars/language-jolie:
 vendor/grammars/language-jsoniq:
 - source.jq
 - source.xq
+vendor/grammars/language-kotlin:
+- source.Kotlin
```

Comment:
**Nitpick:** By convention, this should ideally be lowercase:

~~~
source.kotlin
~~~

Not a huge issue, but certainly a nice-to-have. Note that you'll need to update [this line](https://github.com/nishtahir/language-kotlin/blob/f5658d73ae84ac05369c2fe5e53232e2c3301c88/syntax/Kotlin.tmLanguage#L1293) in your grammar as well:

~~~xml
     <key>scopeName</key>
-    <string>source.Kotlin</string>
+    <string>source.kotlin</string>
     <key>uuid</key>
     <string>d508c059-a938-4779-b2bc-ff43a5078907</string>
~~~